ie-solver/ie_mat.cpp

```diff
@@ -444,13 +443,15 @@ int ie_Mat::id( std::vector<unsigned int>& p, ie_Mat& Z, double tol) const {
 }
 
 
-// TODO fix this mess
 void ie_Mat::print() const{
-	// for(int i=0; i<height_; i++){
-	// 	for(int j=0; j<width_; j++){
-	// 		sprintf("%.7f ",get(i,j));
-	// 	}sprintf("\n");
-	// }
+	std::string message = "\n";
+	for(unsigned int i=0; i<height_; i++){
+		for(unsigned int j=0; j<width_; j++){
+			message += std::to_string(1000*get(i,j)) + " ";
```

Comment:
If we're really going with this scheme to avoid all the 0s printed (can't we do some kind of scientific notation?), at least log beforehand that that's what's happening